### PR TITLE
Switch to brightv1.1 in leaderboard

### LINKED ISSUE #4311: Change default reasoning benchmark from BRIGHT(v1) to BRIGHT(v1.1)
### Description of the feature

BRIGHT v1.1's implementation is much cleaner and should be the default - we will have to rerun the models using the new implementation though before we can switch over

Discussed here: https://github.com/embeddings-benchmark/mteb/issues/1517#issuecomment-4148235399

### PR DESCRIPTION
**Pull request by Samoed**
Close https://github.com/embeddings-benchmark/mteb/issues/4311

### CODE REVIEW

**Review by KennethEnevoldsen — COMMENTED** [review]
aren't we hereby dropping some models?

**Comment by Samoed** [comment]
I tried to run 7b models as much as possible, but leet code takes too much time. I'll try to run them on Niklas cluster

**Comment by Samoed** [comment]
image

**Comment by KennethEnevoldsen** [comment]
Hmm we are still dropping quite a few models - I see some large models like gritLM, and some APIs like voyage. If we make a list then I can start running some as well

**Comment by Samoed** [comment]
Api
- ByteDance-Seed/Seed1.5-Embedding
- google/text-embedding-004
- voyageai/voyage-large-2-instruct
- openai/text-embedding-3-large
- Cohere/Cohere-embed-english-v3.0

7B
- Alibaba-NLP/gte-Qwen2-7B-instruct
- Alibaba-NLP/gte-Qwen1.5-7B-instruct
- GritLM/GritLM-7B
- Salesforce/SFR-Embedding-Mistral
- intfloat/e5-mistral-7b-instruct
- infly/inf-retriever-v1

Small:
- manveertamber/cadet-embed-base-v1

**Comment by Samoed** [comment]
It took me 4-5 hours to run e5-large on 8xA100 with small hack

```python
model = mteb.get_model("intfloat/e5-base")

class DataParallelHF(torch.nn.DataParallel):
    def __getattr__(self, name):
        try:
            return super().__getattr__(name)
        except AttributeError:
            return getattr(self.module, name)


model.model[0].auto_model = DataParallelHF(
    model.model[0].auto_model
)
```

**Comment by KennethEnevoldsen** [comment]
Is that something that we want to integrate more permanently?

**Comment by KennethEnevoldsen** [comment]
I will start running the APIs

```
ByteDance-Seed/Seed1.5-Embedding
google/text-embedding-004
voyageai/voyage-large-2-instruct
openai/text-embedding-3-large
Cohere/Cohere-embed-english-v3.0
```

**Comment by Samoed** [comment]
> Is that something that we want to integrate more permanently?

I don't think so. I think it would be better to integrate into sentence transformers directly. I saw few changes for this in new sentence transformers version

**Comment by KennethEnevoldsen** [comment]
Thanks for the ping @Samoed - I will start rerunning some results (I will be travelling tomorrow, so will probably set it to run the day thursday)

File: mteb/benchmarks/benchmarks/benchmarks.py [hunk 1]
```diff
@@ -1375,9 +1376,9 @@
 """,
 )
 
-BRIGHT_V1_1 = Benchmark(
+BRIGHT_V1_1 = BRIGHTBenchmark(
     name="BRIGHT(v1.1)",
-    display_name="Reasoning Retrieval",
+    display_name="Reasoning Retrieval v1.1",
```
Comment (KennethEnevoldsen) [inline mteb/benchmarks/benchmarks/benchmarks.py:1381]: Maybe it is more clear to simply call it BRIGHT?

```suggestion
    display_name="Reasoning Retrieval",
```

**Review by KennethEnevoldsen — COMMENTED** [review]
Starting to run some of these models (have been svamped with other things, but it is still on my todo).

File: mteb/benchmarks/benchmarks/benchmarks.py (hunk 1 above)
Comment (Samoed) [reply mteb/benchmarks/benchmarks/benchmarks.py:1381]: Our display name should be unique. How we should call v1 version? `Reasoning Retrieval (legacy)`?

**Comment by Samoed** [comment]
I'll also try to setup Niklas server to evaluate some models

**Comment by Samoed** [comment]
We still need to update

**Comment by KennethEnevoldsen** [comment]
Yea, it is on my todo, just not at the top (some of the runs are already submitted, we just need to merge them, but there is a few issue).

We could remove the files with issues and merge and then do another round.
